src/pages/Add2/Add2.jsx

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import FileBase from 'react-file-base64';
+/* import axios from 'axios'; */
+
+import './Add2.css';
+import './loader.css';
+import { useNavigate } from 'react-router-dom';
+
+import { addDoctor, finishDoctor } from '../../redux/doctors/adds';
+
+export default function Add2() {
+  /* Load Redux State */
+  const adds = useSelector((state) => state.add);
+
+  /* Setup Redux dispatch */
+  const dispatch = useDispatch();
+
+  const history = useNavigate();
+
+  // *********************************
+
+  const [listingData, setListingData] = useState('');
+  const [ansApi, setAnsApi] = useState('');
+  const [isLoader, setIsLoader] = useState(false);
+
+  let data64; // init variable
+
+  // console.log(listingData)
```

Comment:
- [ ] Kindly, remove this old piece of commented out code in order to improve the repo's cleanliness. 👍